src/main/java/at/hannibal2/skyhanni/test/TestExportTools.kt

```diff
@@ -47,18 +47,23 @@ object TestExportTools {
 
     @SubscribeEvent
     fun onKeybind(event: GuiScreenEvent.KeyboardInputEvent.Post) {
-        if (!OSUtils.isKeyHeld(SkyHanniMod.feature.dev.copyNBTDataCompressed)) return
+        if (!OSUtils.isKeyHeld(SkyHanniMod.feature.dev.copyNBTDataCompressed) || !OSUtils.isKeyHeld(SkyHanniMod.feature.dev.copyNBTData) ) return
         val gui = event.gui as? GuiContainer ?: return
         val focussedSlot = gui.slotUnderMouse ?: return
         val stack = focussedSlot.stack ?: return
-        val json = toJson(Item, stack)
-        OSUtils.copyToClipboard(json)
-        LorenzUtils.chat("Copied test importable to clipbooard")
+        if (!OSUtils.isKeyHeld(SkyHanniMod.feature.dev.copyNBTDataCompressed)){
```

Comment:
I think it'd be more useful if you reused CopyItemCommand.command instead since it has more data and is easier to read, more akin to the SBA copy nbt feature. Could maybe refactor the code out to a generic method that takes an item/stack and use it in both places

Reply:
Yeah, fair. Was being lazy, will take a look at some point